modules/ROOT/pages/cors-policy.adoc

```diff
@@ -4,24 +4,49 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cors, api gateway, gateway, policy
 
-Cross-Origin Resource Sharing (CORS) is a mechanism by which a web application can access resources defined in another domain. Browsers for example, rely on this standard for allowing cross origin requests.
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | cross-origin resource sharing (CORS)
+>s| Summary      | Enables access to resources residing in external domains.
+>s| Category | Compliance
+>s| Available from Mule version | v4.1.1
+.4+>.^s| Returned Status Codes |
 
-The CORS policy complies with Cross-Origin Resource Sharing W3C Recommendation 16 January 2014. From API Manager, you can apply the policy, the default policy configuration considers the API to be a public resource. If not, removing the Public Resource check will enter the extended CORS configuration.
+CORS is a mechanism by which a Web application can access resources that are defined in another domain.  Using CORS, applications invoking JavaScript XMLHttpRequest (XHR) calls from a Web page can interact with resources from non-origin domains.
+
+Typically, Web browsers implement this standard for allowing cross origin requests. Your API is categorized as a public resource by default when you apply this policy. If you want to share your API with only specific resources, you can configure the policy as a restricted resource.
+
+The CORS policy complies with CORS W3C recommendation from 16 January 2014.
+
+*can we add a link to it?*
+
+*<Eva: I saw that you can specify a default group in the UI. What is it? what does it do and what other groups can one create?>*
+
+You must specify the default group before specifying another one. The Default group is not a fallback in the normal sense of a default. In this case, it is only the first group you configure for CORS. -- Eva, not sure about this sentence. What does the fallbacl mean here. Please clarify.
+
+You cannot order the CORS policy. This isn't a technical limitation, but rather by design. No OPTIONS request can reach the backend if CORS is configured, as this request is treated as a Preflight according to the CORS-compliant RFC.
+
+
+
+*Eva again the previous sentence does not make sense*
 
 image::cors-policy.png[]
 
-NOTE: You must specify the default group before specifying another one. The Default group is not a fallback in the normal sense of a default. In this case, it is only the first group you configure for CORS.
+*Eva--do we have any prerequisites or scope and considerations for this policy?*
```

Comment:
What do you mean by pre requisites or scope? If you mean some third party dependency as in the OAuth policies, then no. You can always apply this policy to a tracked API.

Reply:
Ok! Thanks!